lib/solenoids/solenoids.cpp

```diff
@@ -322,6 +358,9 @@ namespace Solenoids {
   }
 
   int armPropane() {
+    digitalWrite(lox_gems_pin, 1);
```

Comment:
we should change these names

Reply:
that is quite a rabbit hole to go down, given all of our "propane" names scattered throughout lol. Maybe on the next merge though